src/index.html

```diff
@@ -4,9 +4,477 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./styles/index.scss">
+    <script src="./scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header header--sticky">
+      <a href="#" class="logo">
+        <img
+          class="logo__brand-img"
+          src="images/logo.png"
+          alt="moyo logo"
+        >
+      </a>
+
+      <nav class="nav nav--fixed">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="burger">
+        <span class="burger__dash burger__dash--one"></span>
+        <span class="burger__dash burger__dash--two"></span>
+        <span class="burger__dash burger__dash--three"></span>
+      </div>
+    </header>
+
+    <main class="main">
+      <div class="cards">
+        <div class="cards__row">
```

Comment:
I suggest getting used to semantic elements. You can wrap all of your card into `<section class="cards">`, cause this is actually a product section, and your card in `<article class="card">`

<img width="626" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/49764881/ce0cdd58-e464-4d68-99aa-dbcbcc33902e">